docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+Studio is a tool to develop AsyncAPI documentation. 
```

Comment:
We could improve that description, It's not just about AsyncAPI documentation.

A `Studio` refer to a development environment / platform, or a collection of tools that enable a certain capability.
IMO I'd define studio as an "AsyncAPI development platform (or environment) that enable users to create, visualise, collaborate on event-driven architectures"

@alequetzalli or @fmvilas might have a better description